resources/repo.rb

```diff
@@ -20,7 +20,7 @@
 
 resource_name 'yum_mysql_community_repo'
 
-description 'Configure yum repo for MySQL Community edition'
+description 'Configure yum repo for MySQL Community edition' if description
```

Comment:
I think this is more what you want:

```suggestion
description 'Configure yum repo for MySQL Community edition' if respond_to?(:description)
```